### PR TITLE
Bump template version to include multipart form data fix

### PR DESCRIPTION
**Pull request by Andy-Grigg**
Bump swagger.json to MI 24.2.770, and update codegen template to support file upload and download.

### CODE REVIEW

File: ansys-grantami-serverapi-openapi/poetry.lock [hunk 1]
```diff
@@ -373,21 +373,21 @@ yaml = ["ruamel.yaml"]
 
 [[package]]
 name = "pytest"
-version = "8.0.0"
+version = "7.4.4"
```
Comment (ludovicsteinbach) [inline ansys-grantami-serverapi-openapi/poetry.lock:376]: Any reason for downgrading pytest?

**Review by ludovicsteinbach — COMMENTED** [review]
Changes seem reasonable. Are there some formatting changes in the API spec? Can't see the diff in GitHub because it's too large

File: ansys-grantami-serverapi-openapi/poetry.lock (hunk 1 above)
Comment (Andy-Grigg) [reply ansys-grantami-serverapi-openapi/poetry.lock:376]: The template defines pytest as ^7.0.0, this needs to be bumped to ^8.0.0 to allow us to use a later version. I'll make this change, release a new template, and update this PR.

**Comment by Andy-Grigg** [comment]
> Changes seem reasonable. Are there some formatting changes in the API spec? Can't see the diff in GitHub because it's too large

It looks like the line endings have changed, probably because I downloaded it using Firefox (which reformatted it), and then I fixed the formatting in PyCharm. I'll try and fix this as well.

File: ansys-grantami-serverapi-openapi/poetry.lock (hunk 1 above)
Comment (Andy-Grigg) [reply ansys-grantami-serverapi-openapi/poetry.lock:376]: https://github.com/ansys/openapi-client-template/pull/152

**Comment by Andy-Grigg** [comment]
This PR has gotten really messy, I'm going to close it and create a new one.

**Pull request closed without merging**